cloud/linode/service_controller.go

```diff
@@ -40,7 +42,7 @@ func (s *serviceController) Run(stopCh <-chan struct{}) {
 				return
 			}
 
-			if service.Spec.Type != "LoadBalancer" {
+			if service.Spec.Type != specType {
```

Comment:
You can use the const from the core/v1 package already imported up top for this and the others:
```
if service.Spec.Type != v1.ServiceTypeLoadBalancer
```